e2e/app/component/runtime-panel.ts

```diff
@@ -0,0 +1,192 @@
+import Container from 'app/container';
+import {LinkText} from 'app/text-labels';
+import Button from 'app/element/button';
+import {Page} from 'puppeteer';
+import {waitForAttributeEquality, waitWhileLoading} from 'utils/waits-utils';
+import PrimereactInputNumber from 'app/element/primereact-input-number';
+import SelectMenu from "./select-menu";
+import {savePageToFile, takeScreenshot} from "../../utils/save-file-utils";
+import BaseElement from "../element/base-element";
+
+const defaultXpath = '//*[@id="runtime-panel"]';
+const startStopIconXpath = '//*[@data-test-id="runtime-status-icon"]';
```

Comment:
nit: `statusIconXpath` seems more accurate? or at least consistent with the UI name.